src/main/java/org/simulator/fba/FluxBalanceAnalysis.java

```diff
@@ -418,4 +450,22 @@ private double stoichiometry(SpeciesReference specRef) throws SBMLException, Mod
 		return Double.NaN;
 	}
 
+	/**
+	 * Gets the value of the EPSILON
+	 *
+	 * @return the epsilon value
+	 */
+	public double getEpsilon() {
+		return EPS;
```

Comment:
Please make this lowercase and call it `epsilon` or `eps`.

Reply:
@matthiaskoenig is here referring to the name of the variable, not the name of the method.

Reply:
Updated!